examples/1D_poly_bubscreen/case.py

```diff
@@ -106,6 +106,8 @@
     'time_stepper'                 : 1,
     'weno_order'                   : 1,
     'weno_eps'                     : 1.E-16,
+    'weno_Re_flux'                 : 'T',  
```

Comment:
this example does not have viscosity, make these both 'F'

Reply:
Hi @sbryngelson, thanks for pointing out it. It's resolved.